plugins/package-managers/gleam/src/main/kotlin/Gleam.kt

```diff
@@ -41,6 +41,7 @@ import org.ossreviewtoolkit.utils.common.div
 private const val GLEAM_TOML = "gleam.toml"
 private const val MANIFEST_TOML = "manifest.toml"
 internal const val PACKAGE_TYPE_OTP = "OTP"
+internal const val PACKAGE_TYPE_HEX = "Hex"
```

Comment:
I somewhat do get why "Gleam" was extracted separately from "OTP", but doing "Hex" yet separately seems like being too detailed. How about squashing this to the previously commit saying "Extract remaining source types to constants"?